lib/ansible/modules/network/fortios/fortios_config.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/python
+#
+# Ansible module to manage configuration on fortios devices
+# (c) 2016, Benjamin Jolivot <bjolivot@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+ANSIBLE_METADATA = {
+    'status': ['preview'],
+    'supported_by': 'community',
+    'version': '1.0'
+}
+
+DOCUMENTATION = """
+---
+module: fortios_address
+version_added: "2.3"
+author: "Benjamin Jolivot (@bjolivot)"
+short_description: Manage fortios firewall config
+description:
+  - This module provide management of FortiOS Devices configuration.
+options:
+  src:
+    description:
+      - target configuration file path.
+  dest:
+    description:
+      - Backup configuration file path.
+  filter:
+    description:
+      - in case of backup, if you want partial config, you can restrict by giving expected configuration path (ex. firewall address).
+    default: ""
+  state:
+    description:
+      - Specify if address need to be added or deleted.
+    required: true
+    choices: ['updated', 'backuped']
+
+notes:
+  - This module requires pyFG python library
+"""
+
+EXAMPLES = """
+- name: Backup current config
+  fortios_config:
+    host: 192.168.0.254
+    username: admin
+    password: password
+    state: backuped
+    dest: ./backup.conf
+
+- name: Backup only address objects, log all commands to file
+  fortios_config:
+    host: 192.168.0.254
+    username: admin
+    password: password
+    state: backuped
+    dest: ./backup_address_objects.conf
+    filter: "firewall address"
+    logfile: fortios.log
+
+- name: Update configuration from file
+  fortios_config:
+    host: 192.168.0.254
+    username: admin
+    password: password
+    state: updated
+    src: new_configuration.conf
+
+"""
+
+RETURN = """
+running_config:
+  description: full config string
+  returned: always
+  type: string
+change_string:
+  description: The commands really executed by the module
+  returned: only if config changed
+  type: string
+"""
+import logging
```

Comment:
please remove logging, it is seen as potential leak of sensitive information. To return data e.g. commands used back to the user, add them to exit_json() and document with return.